app/src/androidTest/java/com/example/android/roomwordssample/WordDaoTest.java

```diff
@@ -51,7 +49,7 @@
 
     @Before
     public void createDb() {
-        Context context = InstrumentationRegistry.getTargetContext();
+        Context context = getInstrumentation().getTargetContext();
```

Comment:
didn't this show as deprecated?

Use ApplicationProvider.getApplicationContext()